src/ViewItem.js

```diff
@@ -80,7 +80,7 @@ class ViewItem extends React.Component {
     // the top item, sorted by loan-date descending, is a best-effort.
     loans: {
       type: 'okapi',
-      path: 'circulation/loans?query=(itemId==!{itemId}) sortby loanDate/sort.descending&limit=1',
+      path: 'circulation/loans?query=(itemId==!{itemId})&limit=1 sortby loanDate/sort.descending',
```

Comment:
@mkuklis 
![image](https://user-images.githubusercontent.com/8953703/50608819-bf142900-0e9b-11e9-8ca2-b66466d7e646.png) I receive an error popup failing to fetch the loans here.

Reply:
Good catch! Query parameters can/should be specified separately from the path, e.g. https://github.com/folio-org/ui-users/blob/aa9e67298b60ef10e68df6e5886d734c87ea2b62/src/Users.js#L83-L91

Reply:
@mkuklis It happened when I pulled down the branch and ran it locally.